ChangeLog

```diff
@@ -102,6 +102,10 @@ What's New in Pylint 1.8?
 
       Close #574
 
+    * Check for string concatenation inside a list, introducing new
+      W1403 message (patch by Lucas Cimon)
```

Comment:
I'd really prefer to see `message_name` instead of `Wxxxx`

Reply:
OK